### ISSUE TITLE
cull-entities error

### ISSUE THREAD

**Issue by LogGits** [VolmitSoftware/React#346]
Error when using the command `/cmi act cull-entities @e`. I know this is improper usage but it should be handled. It is returning this error:
```
[05:39:22 WARN]: com.volmit.react.api.SelectorParseException: MISSING ":" All params must start with @<key>:<opts>
[05:39:22 WARN]: 	at com.volmit.react.api.Selector.createSelector(Selector.java:65)
[05:39:22 WARN]: 	at com.volmit.react.command.CommandAct.fire(CommandAct.java:163)
[05:39:22 WARN]: 	at com.volmit.react.controller.CommandController.onCommand(CommandController.java:318)
[05:39:22 WARN]: 	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44)
[05:39:22 WARN]: 	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:151)
[05:39:22 WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:685)
[05:39:22 WARN]: 	at net.minecraft.server.v1_12_R1.PlayerConnection.handleCommand(PlayerConnection.java:1472)
[05:39:22 WARN]: 	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1277)
[05:39:22 WARN]: 	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:45)
[05:39:22 WARN]: 	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:5)
[05:39:22 WARN]: 	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:14)
[05:39:22 WARN]: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[05:39:22 WARN]: 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[05:39:22 WARN]: 	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46)
[05:39:22 WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:843)
[05:39:22 WARN]: 	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:426)
[05:39:22 WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:767)
[05:39:22 WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:665)
[05:39:22 WARN]: 	at java.lang.Thread.run(Thread.java:748)
```

**Comment by cyberpwnn** [comment]
Use @e:pig 
Or @e:pig&cow 

Use /re act and hover over the @e it shows you how to use it

**Comment by LogGits** [comment]
@cyberpwnn I am well aware of its usage. I just wanted to inform you that it is spewing out errors in console. These errors should be caught. Errors make a plugin look faulty. I know that the error suggests im using the command incorrectly, however, it shouldn't spew out the error to console.

**Comment by cyberpwnn** [comment]
It's not an error, it's called an exception. 

Every time a player disconnects from the game, the server throws an exception. It's not an error, and the server isnt faulty for it. 

Sure, ill catch the exception, but this is a cosmetic fix.

**Comment by cyberpwnn** [comment]
Fixed in 6.541
